apps/vscode-wing/src/console/console-manager.ts

```diff
@@ -230,6 +237,7 @@ export const createConsoleManager = (
       webviewPanel.onDidDispose(async () => {
         resourcesExplorer.clear();
         testsExplorer.clear();
+        endpointsExplorer.clear();
```

Comment:
rebase from `main`

Reply:
Will make the changes asap, sorry forgot what merge ff does to the git history